601-spring-data-primitive-types/src/main/java/io/quarkus/qe/spring/data/BookRepository.java

```diff
@@ -24,6 +26,6 @@
     @Query(value = "SELECT b.isbn FROM Book b where b.bid = :bid")
     Long customFindPublicationIsbnObject(@Param("bid") Integer bid);
 
-    // issue QUARKUS-525
+    // issue 13015
```

Comment:
Please, add "TODO" and the fully qualified url of the issue, this way is much easier to search.

Reply:
This `issue` is just a reference in order to track, what this test is trying to cover. Is not a TODO. 
I will change `issue` by `githubRef`.

Reply:
I don't really like to have links or github references to things that are already resolved.
Everything we have in the tests are motivated by either an issue/feature, so we don't need to individually track any code snippets.
What do you think? cc @mjurc

Reply:
I think that our scenarios are sometimes very weird, and are easy to understand if you have a reference to the issue that is covering. Is like your context. Even more, if we use one scenario to cover different issues along the time.

Reply:
For example, could you tell me what is wrong, or what this test is trying to accomplish on this scenario:

` List<Book> findByPublisherAddressZipCode(@Param("zipCode") String zipCode);` 

This line is covering [a specific case](https://github.com/quarkusio/quarkus/pull/13015). 

If we drop the reference, somebody could change this line or some other stuff from the scenario, and maybe the test that is behind will also pass with the new "scenario" because the result is the same. But the aim of the test is totally broken.

Reply:
I don't have a strong opinion about their removal. Maybe some more obvious line to, well, make it more obvious it's a regression check for an issue?

Like, eg, `// This is a regression check for https://www.myissue`.

Reply:
I think this may be especially useful in tests which may otherwise not make much sense, actually.